client_plugin/utils/fs/device_watcher_windows.go

```diff
@@ -74,20 +75,23 @@ func (w *DeviceWatcher) Init() {
 // awaitEventAndEmit awaits and emits a device change event or an error.
 func (w *DeviceWatcher) awaitEventAndEmit() {
 	script := fmt.Sprintf(devChangeWaitScript, devChangeTimeoutSec)
-	out, err := exec.Command(powershell, script).CombinedOutput()
+	stdout, stderr, err := ps.Exec(script)
 	if err != nil {
-		log.WithFields(log.Fields{"err": err, "out": string(out)}).Error("Failed to watch device event ")
+		log.WithFields(log.Fields{"err": err, "stdout": stdout,
+			"stderr": stderr}).Error("Failed to watch device event ")
 		select {
 		case w.Error <- err:
-			log.WithFields(log.Fields{"err": err, "out": string(out)}).Info("Successfully emitted error ")
+			log.WithFields(log.Fields{"err": err, "stdout": stdout,
+				"stderr": stderr}).Info("Successfully emitted error ")
 		default:
-			log.WithFields(log.Fields{"err": err, "out": string(out)}).Warn("Couldn't emit error, continuing.. ")
+			log.WithFields(log.Fields{"err": err, "stdout": stdout,
```

Comment:
There's a `err != nil` check, and it would print `nil` even if that wasn't in place.